src/input/Mouse.cc

```diff
@@ -128,14 +132,22 @@ uint8_t Mouse::read(EmuTime::param /*time*/)
 {
 	if (mouseMode) {
 		switch (phase) {
-		case PHASE_XHIGH:
+		case PHASE_XHIGH1:
 			return ((xRel >> 4) & 0x0F) | status;
-		case PHASE_XLOW:
+		case PHASE_XLOW1:
 			return  (xRel       & 0x0F) | status;
-		case PHASE_YHIGH:
+		case PHASE_YHIGH1:
 			return ((yRel >> 4) & 0x0F) | status;
-		case PHASE_YLOW:
+		case PHASE_YLOW1:
 			return  (yRel       & 0x0F) | status;
+		case PHASE_XHIGH2:
+		case PHASE_XLOW2:
+		case PHASE_YHIGH2:
+		case PHASE_YLOW2:
+			// Keep the delta zero in the alternate cycle of the scan round
+			// to avoid trackball detection for large (absolute) values
+			// Timeout resets the state to YLOW2 so that each scan round syncs to XHIGH1
+			return status;
```

Comment:
I agree with this 1st commit. Though I propose to simplify the implementation a bit.
I propose to implement this function as:
```
switch (phase) {
    case PHASE_XHIGH1: case PHASE_XHIGH2:
        ...
    case PHASE_XLOW1: case PHASE_XLOW2:
        ...
    case PHASE_YHIGH1: case PHASE_YHIGH2:
        ...
    case PHASE_YLOW1: case PHASE_YLOW2:
        ...
}
```
So do exactly the same for the '1' and '2' phases. Instead we'll implement the different behavior in write(). That keeps the trick (=clamp with workaround for trackball detection) more localized.